### DISCUSSION TITLE
The Gopro 10 black camera stopped working after Gopro Labs.

### DISCUSSION THREAD

**Discussion by Kocik001** [gopro/labs#826] [General]
Greetings forumchane, I have a 10 black camera, I flashed it in gopro Labs, used it everything was OK, but when I wanted to install the standard firmware, it hung and no longer works(Now it does not turn on, it only blinks with the front screen (It does not create folders on the flash card (How can I revive it? I'm asking for help.

**Comment by dnewman-gpsw** [comment]
Can you add details, installing firmware is not typically a cause of a failure (with millions of GoPro firmware install per year.)  What things did you do after the install?

**Reply by Kocik001** [reply]
Thank you for responding, the camera worked well with the Gopro Labs firmware, but at the time of installing the standard firmware, the camera froze and did not turn on anymore (I even tried to revive it on your advice with the 11 Gopro you specified here on the forum, the camera starts blinking diodes but still does not turn on(((

**Reply by dnewman-gpsw** [reply]
Removed the 11 recovery procedure, as that will not work on a 10 as is. 

Have you tried reinstalling Labs?  Does any install firmware procedure start?

**Reply by Kocik001** [reply]
I tried to flash it, but due to the fact that the camera does not turn on, the diodes do not blink, nothing worked ( factory_reset.txt  I also tried(

**Reply by Kocik001** [reply]
Gopro labs I also tried it, too, without results (

**Reply by dnewman-gpsw** [reply]
What stock version were you installing when it failed? There are some older versions that you can't roll back to.

**Reply by Kocik001** [reply]
1.20

**Reply by Kocik001** [reply]
after gopro labs, I wanted to set 1.20 but it didn't work out and the camera froze and didn't turn on anymore(

**Reply by dnewman-gpsw** [reply]
That version can no longer be installed, although I'm not aware of it bricking, I would have thought it just wouldn't install anymore. You might have to try regular support as I don't know what happens going from V1.5 to V1.2, this is completely independent of Labs.  Something changed around V1.4x I think.  I don't have any spare H10s to test this, but you can try this updated for H10 recovery (same procedure at H11, but different data) https://www.dropbox.com/scl/fi/2n8tlv35z3ts5xpbno2os/RecoveryHERO10v150.zip?rlkey=zlfxmirmwzk6phnccra8bc7zl&dl=1

**Reply by Kocik001** [reply]
Unfortunately, nothing worked out, the camera starts blinking the front diode many times, then it just stays on or goes out (((((Thank you for your help, but it doesn't seem to be revived(((

**Reply by retlw** [reply]
> Unfortunately, nothing worked out, the camera starts blinking the front diode many times, then it just stays on or goes out (((((Thank you for your help, but it doesn't seem to be revived(((

If you still have a camera, try replacing 0x40200000 in the pardata.par file with 0x40080000 (both)

**Reply by HighwayStar** [reply]
Does someone still have link to recovery files? I've bricked mycmeratoo trying to downgrade firmware and all recovery links are dead.
